openapi2beans/JavaChecker/src/main/java/dev/galasa/openapi2beans/example/generated/BeanToTestArraysWithVariousPrimitiveTypes.java

```diff
@@ -0,0 +1,43 @@
+package dev.galasa.openapi2beans.example.generated;
+
+// A bean to test array property can contain any primitive property type
+public class BeanToTestArraysWithVariousPrimitiveTypes {
+    // Class Variables //
+    private boolean[] aBooleanArray;
+    private int[] anIntArray;
+    private double[] aNumberArray;
+    private String[] aStringArray;
+
+    // Constants //
+
+    public BeanToTestArraysWithVariousPrimitiveTypes () {
```

Comment:
I think we will need some parameters to appear in the constructor.

Reply:
Added different test yaml for this